src/translations/lang/en.json

```diff
@@ -99,6 +99,8 @@
 		"ANNOUNCEMENT_MESSAGE": "MyCrypto.com no longer allows the use of private keys, mnemonics, or keystore files in the browser. To continue using them, please download the [MyCrypto Desktop App](https://download.mycrypto.com).",
 		"APPROVE_MEMBERSHIP": "Approve Membership Contract",
 		"APPROVE_SWAP": "Approve Swap",
+		"APP_LOADING": "Loading...",
```

Comment:
Why is this part of this PR?
- Same for many more cases in the file

Reply:
see https://github.com/MyCryptoHQ/MyCrypto/pull/3318#discussion_r435498392